<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>105</y>
      <width>111</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>49</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>BASE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>NÃšMERO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Num">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>63</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Resposta:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Executar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PT Root UI Light</family>
      <pointsize>12</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>EXECUTAR </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_Resultado">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Terminal</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
